core/native.gradle

```diff
@@ -30,14 +30,8 @@ buildscript {
     }
 }
 
+
 import org.apache.commons.io.FilenameUtils
-import org.gradle.nativeplatform.BuildType
-import org.gradle.nativeplatform.NativeBinarySpec
-import org.gradle.nativeplatform.NativeLibrarySpec
-import org.gradle.nativeplatform.SharedLibraryBinarySpec
-import org.gradle.nativeplatform.StaticLibraryBinarySpec
-import org.gradle.nativeplatform.platform.NativePlatform
-import org.gradle.nativeplatform.toolchain.VisualCpp
```

Comment:
These are used below. Removing this breaks the script.

Reply:
ah okay didn't carefully looked at it after checkstyle ran

Reply:
Fixed